src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -136,9 +140,13 @@ class ClusterMonitorActor(val cluster: Cluster,
     // Only do this if the cluster was not created with the pet service account.
     val bucketACLFuture = setStagingBucketACLsForUser()
 
+    implicit val timeout: Timeout = Timeout(5 seconds)
+    val enableClusterProxying: Future[Any] = clusterDnsCache ? ProcessReadyCluster(cluster)
```

Comment:
Proxyable = (host IP is in the cache)

We don't set the cluster to Running until after this is true.